src/commands/sourcemaps/renderer.ts

```diff
@@ -32,8 +35,67 @@ export const renderRetriedUpload = (payload: Payload, errorMessage: string, atte
   return chalk.yellow(`[attempt ${attempt}] Retrying sourcemap upload ${sourcemapPathBold}: ${errorMessage}\n`)
 }
 
-export const renderSuccessfulCommand = (fileCount: number, duration: number) =>
-  chalk.green(`${ICONS.SUCCESS} Uploaded ${fileCount} files in ${duration} seconds.\n`)
+export const renderSuccessfulCommand = (statuses: UploadStatus[], duration: number, dryRun: boolean) => {
+  const results = new Map<UploadStatus, number>()
+  statuses.forEach((status) => {
+    if (!results.has(status)) {
+      results.set(status, 0)
+    }
+    results.set(status, results.get(status)! + 1)
+  })
+
+  const output = ['', chalk.bold('Command summary:')]
+  if (results.get(UploadStatus.Failure)) {
+    output.push(chalk.red(`${ICONS.FAILED} All sourcemaps have not been uploaded correctly.`))
```

Comment:
Small nit: I find `Some sourcemaps have not been uploaded correctly` a bit clearer and less cumbersome that `All sourcemaps have not..` - WDYT ?

Reply:
Indeed, I updated this using your sentence, thank you!